.github/workflows/codacy-coverage-reporter.yaml

```
@@ -0,0 +1,13 @@
+name: Codacy Coverage Reporter\
+on: "push"
+jobs:
+  codacy-coverage-reporter:
+    runs-on: ubuntu-latest
+    name: codacy-coverage-reporter
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)